tests/checkpoint/test_checkpoint.py

```diff
@@ -661,13 +672,16 @@ def test_checkpoint_configuration_warning_error_quarantine_test_yaml_config(
     assert len(data_context.validations_store.list_keys()) == 2
     assert result.success
 
+    monkeypatch.delenv("GE_ENVIRONMENT")
+
```

Comment:
This elegance due to `monkeypatch` was courtesy of @Aylr !

Reply:
@roblim Sorry, GitHub is being unclear: To "nix" what?  Not sure if it is OK to nix `monkeypatch.delenv("GE_ENVIRONMENT")` if that is what you mean.  Please let me know.  Thanks.

Reply:
Oh, sorry. I thought Taylor had mentioned somewhere that all the `monkeypatch.delenv` at the end of test functions was unnecessary since the monkeypatch mock env was scoped per function. Must have misread.